packages/ia-components/sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx

```diff
@@ -268,7 +269,11 @@ class AudioPlayerWithYoutubeSpotify extends Component {
             source={channelToPlay}
             backgroundPhoto={itemPhoto}
             sourceData={this.getAudioSourceInfoToPlay()}
-            customSourceLabel={getChannelLabelToDisplay({ channel: channelToPlay, labelValue: audioPlayerChannelLabel })}
+            customSourceLabel={getChannelLabelToDisplay({
+              channel: channelToPlay,
+              labelValue: audioPlayerChannelLabel,
+              title: `playing ${audioPlayerChannelLabel}`
```

Comment:
QUESTION: Why is this title in the present progressive "playing" instead of "play"? Is it because this is not a button action but a labeling of current status?

Reply:
Correct. this instance is the tab beneath the content viewer where the player resides